packages/pdf-viewer/package.json

```diff
@@ -9,11 +9,17 @@
     "access": "restricted"
   },
   "scripts": {
+    "tsc": "",
     "tsc_DISABLED": "tsc --project tsconfig.json",
```

Comment:
The empty `"tsc"` seems to be required to prevent `yarn` from running `tsc` anyway (presumably using the version of `tsc` from the `typescript` package) when running `yarn workspaces foreach ... tsc`.